modules/Hoot/ui/managePanel/TaskingManagerPanel.js

```diff
@@ -0,0 +1,373 @@
+        this.projectList = Hoot.api.getTMProjects();
+        this.createTables();
+
+        return this;
+    }
+
+    async activate() {
+        // await for the list
+        this.projectList = await this.projectList;
+
+        if ( this.projectList ) {
+            this.projectList.features = this.projectList.features.filter( task => task.properties.hoot_map_id );
+
+            this.loadProjectsTable( this.projectList.features );
+        }
+    }
+
+    createTables() {
+        const projectsDiv = this.panelWrapper.append( 'div' )
+            .classed( 'taskingManager-projects', true );
+        projectsDiv.append( 'h3' )
+            .text( 'Tasking Manager Projects' );
+
+        this.projectsTable = projectsDiv.append( 'div' )
+            .classed( 'taskingManager-table keyline-all', true );
+
+        // Initially hidden
+        this.tasksContainer = this.panelWrapper.append( 'div' )
+            .classed( 'taskingManager-tasks hidden', true );
+        this.tasksContainer.append( 'div' )
+            .classed( 'taskHeader', true )
+            .append( 'h3' )
+            .classed( 'taskHeader-title', true );
+
+        this.tasksTable = this.tasksContainer.append( 'div' )
+            .classed( 'taskingManager-table tasks-table keyline-all', true );
+    }
+
+    loadProjectsTable( projects ) {
+        let items = this.projectsTable.selectAll( '.taskingManager-item' )
+            .data( projects );
+
+        items.exit().remove();
+
+        items = items.enter()
+            .append( 'div' )
+            .attr( 'id', d => d.id )
+            .classed( 'taskingManager-item fill-white keyline-bottom', true );
+
+        let header = items
+            .append( 'div' )
+            .classed( 'taskingManager-header flex justify-between align-center', true );
+        header.append( 'div' )
+            .classed( 'taskingManager-title', true )
+            .append( 'a' )
+            .text( project => project.properties.name )
+            .on( 'click', project => {
+                this.loadTaskTable( project );
+            } );
+
+        let body = items.append( 'div' )
+            .classed( 'taskingManager-body', true );
+
+        let description = body.append( 'div' )
+            .classed( 'taskingManager-description', true );
+        description.append( 'label' )
+            .text( 'Description: ' );
+        description.append( 'span' )
+            .text( project => project.properties.short_description );
+
+        let details = body.append( 'div' )
+            .classed( 'taskingManager-details', true );
+        details.append( 'label' )
+            .text( 'Created At: ' );
+        details.append( 'span' )
+            .text( project => {
+                let { created, author } = project.properties;
+                created = new Date( created ).toLocaleString();
+
+                return `${ created } by ${ author }`;
+            } );
```

Comment:
Let's make this look more like TM with the project # in the title, no label for `Description:` and a matching layout for created by and created on.
![image](https://user-images.githubusercontent.com/2600932/85900093-34309800-b7cd-11ea-9155-109b993bcaef.png)

No need for the last updated, priority, map or percent complete.